gcp/api/v1/README.md

```diff
@@ -25,16 +25,26 @@ To deploy the service proxy,
 
 ### Test
 ```
-../deploy_service_proxy api-test.osv.dev osv-grpc-v1-test api_config_test.yaml
+../deploy_service_proxy oss-vdb api-test.osv.dev osv-grpc-v1-test /path/to/api_config_test.yaml
 ```
 
 ### Staging
 ```
-../deploy_service_proxy api-staging.osv.dev osv-grpc-v1-staging /path/to/api_config_staging.yaml
+../deploy_service_proxy oss-vdb api-staging.osv.dev osv-grpc-v1-staging /path/to/api_config_staging.yaml
 ```
 
 ### Production
 
 ```
-../deploy_service_proxy api.osv.dev osv-grpc-v1 /path/to/api_config.yaml
+../deploy_service_proxy oss-vdb api.osv.dev osv-grpc-v1 /path/to/api_config.yaml
 ```
+
+### oss-vdb-test Project
+
+```
+../deploy_service_proxy oss-vdb-test osv-grpc-v1-ivqje5d2hq-uc.a.run.app osv-grpc-v1 /path/to/api_config.yaml
+```
+
+Configuring the Cloud Endpoint requires the Cloud Run service to already exist before running, as it needs the URL.
```

Comment:
Is Cloud Run something that can be managed with terraform as well? Does it make sense to?

Reply:
[There is a terraform resoure for it](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_service).
I'll take a look into it - I'm not sure how updating the container image interacts with terraform.

Reply:
The terraform resource for the cloud run service requires the container image for the service, and changing outside of terraform (as `deploy_service_proxy` does) causes problems.

I suppose we could separate the building and deploying of the container image, and use terraform solely to deploy image versions, but that would be a significant workflow change that can be done in another PR.